library/Fission/Web/User.hs

```diff
@@ -51,6 +56,10 @@ type VerifyRoute
     :> Auth
     :> Verify.API
 
+type VerifyEmailRoute
+  = "verify_email"
+    :> VerifyEmail.API
```

Comment:
Why not on `/user/verify`?

Reply:
`/user/verify` is already taken (although that route is deprecated)
`/user/email/verify` would be fine tho :+1: